### PR TITLE
CPDP-2466: add isFurtherReturn flag

### PR DESCRIPTION
**Pull request by mikail-khan**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala [hunk 1]
```diff
@@ -61,8 +63,14 @@ object DesReturnDetails {
...
-  private def getSource(submitReturnRequest: SubmitReturnRequest): String =
-    submitReturnRequest.agentReferenceNumber.fold("self digital")(_ => "agent digital")
+  private def getSource(submitReturnRequest: SubmitReturnRequest): String = {
+    val source =
+      if (submitReturnRequest.isFurtherReturn)
+        s"${"self digital " + Instant.now.getEpochSecond.toString}"
+      else
+        "self digital"
+    submitReturnRequest.agentReferenceNumber.fold(source)(_ => "agent digital")
+  }
```
Comment (mikail-khan) [inline app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala:73]: does "agent digital" need to have this timestamp added?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala:73]: yeah, we'll need it if an agent did the return as well

File: app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala [hunk 2]
```diff
@@ -61,8 +63,11 @@ object DesReturnDetails {
     )
   }
 
-  private def getSource(submitReturnRequest: SubmitReturnRequest): String =
-    submitReturnRequest.agentReferenceNumber.fold("self digital")(_ => "agent digital")
+  private def getSource(submitReturnRequest: SubmitReturnRequest): String = {
+    val timestamp = Instant.now.getEpochSecond.toString
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala:67]: this should be an empty string if it's not a further return

Comment (mikail-khan) [reply app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala]: fixed

**Comment by mikail-khan** [comment]
@AndyHWChung can you re-review to make sure we are on the same page

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**